developers/documentation/how_to_submit_pr_online.md

```diff
@@ -0,0 +1,101 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# How to open a pull request on GitHub for new documentation
+
+This how-to will guide you to open a pull request in GitHub to contribute your prepared tutorial, how-to or explanation to napari.org. 
+
+This guide is strictly designed for adding new documents - if you want to contribute code to napari check out [this guide instead](../contributing.md).
+
```

Comment:
Oops sorry I missed that one @GenevieveBuckley. What level of subheading were you hoping to go down to in the table? The new theme has `##` headings on the right navigation which scrolls with you e.g.:

![image](https://user-images.githubusercontent.com/17995243/142284170-65cbd304-4601-4afd-8918-d9dcbf918e04.png)

Do you think this would be sufficient?